test/test_config.py

```diff
@@ -9,10 +9,14 @@
 import jsonschema
 import yaml
 
+#TODO: Change to latest schema - 1-0-3? Otherwise use sets for nominal_resolution
 BASE_SCHEMA_URL = "https://raw.githubusercontent.com/ACCESS-NRI/schema"
 BASE_SCHEMA_PATH = "au.org.access-nri/model/output/experiment-metadata"
-DEFAULT_SCHEMA_VERSION = "1-0-0"
-DEFAULT_SCHEMA_COMMIT = "bfc15e3c6fa20d492ccfa0c4706805d64c531e7c"
+SCHEMA_VERSION = "1-0-2"
+SCHEMA_COMMIT = "e9055da95093ec2faa555c090fc5af17923d1566"
+
+LICENSE = "CC-BY-4.0"
+LICENSE_URL = "https://creativecommons.org/licenses/by-sa/4.0/legalcode.txt"
```

Comment:
```suggestion
LICENSE_URL = "https://creativecommons.org/licenses/by/4.0/legalcode.txt"
```
Original URL is the sharealike license.